spec/index.html

```diff
@@ -259,28 +255,100 @@ <h2>A Canonical form of N-Quads</h2>
+    <li>Characters MUST NOT be represented by <code><a href="#grammar-production-UCHAR">UCHAR</a></code>.</li>
     <li>Within <a href="#grammar-production-STRING_LITERAL_QUOTE">STRING_LITERAL_QUOTE</a>,
-      only the characters 
+      the characters 
       <code>U+0022</code>, <code>U+005C</code>, <code>U+000A</code>, <code>U+000D</code>
-       are encoded using <code>ECHAR</code>.
-      <code>ECHAR</code> MUST NOT be used for characters that are
+       MUST be encoded using <code><a href="#grammar-production-ECHAR">ECHAR</a></code>.
+      <code><a href="#grammar-production-ECHAR">ECHAR</a></code> MUST NOT be used for characters that are
       allowed directly in
-      <a href="#grammar-production-STRING_LITERAL_QUOTE">STRING_LITERAL_QUOTE</a>. </li>
-      <li>The token EOL MUST be a single <code>U+000A</code>.</li>
-      <li>The final EOL MUST be provided.</li>
+      <code><a href="#grammar-production-STRING_LITERAL_QUOTE">STRING_LITERAL_QUOTE</a></code>. </li>
+      <li>The token <code><a href="#grammar-production-EOL">EOL</a></code> MUST be a single <code>U+000A</code>.</li>
+      <li>The final <code><a href="#grammar-production-EOL">EOL</a></code> MUST be provided.</li>
   </ul>
-  <div class="issue" data-number="2">
-    Note open errata:
-    <ul>
-      <li><a href="https://www.w3.org/2001/sw/wiki/RDF1.1_Errata#erratum_32">32 – Issues with N-Quads/N-Triples canonicalization</a>.</li>
-      <li><a href="https://www.w3.org/2001/sw/wiki/RDF1.1_Errata#erratum_33">33 – Ambiguity of canonical N-Triples</a></li>
-    </ul>
+
+  <div class="issue" data-number="16">
+    <p>Re-consider the use of `UCHAR` and `ECHAR` escapes in N-Triples/N-Quads canonicalization.
+      The 1.1-based recommendation prohibits the use of `UCHAR` (`U+XXXX`)
+      and allows `ECHAR` only for `U+0022` (quote `\"`),
+      `U+005C` (backslash `\\`),
+      `U+000A` (<code title="LINE FEED"><sub>LF</sub></code> `\n`),
+      and `U+000D` (<code title="CARRIAGE RETURN"><sub>CR</sub></code> `\r`).
+      However, the use of control characters can obfuscate text when presented,
+      creating a potential security concern.</p>
+
+    <p>A future version may consider requiring all characters between
+      `U+0000` and `U+001F` (other than `U+000A` (<code title="LINE FEED"><sub>LF</sub></code>)
+      and `U+000D` (<code title="CARRIAGE RETURN"><sub>CR</sub></code>))
+      to be represented using `UCHAR`.</p>
   </div>
 </section>
 
+<section id="privacy-considerations">
+  <h2>Privacy Considerations</h2>
+  <p class="ednote">TODO</p>  
+</section>
+
+<section id="security">
+  <h2>Security Considerations</h2>
+
+  <p>The <a href="#grammar-production-STRING_LITERAL_QUOTE">STRING_LITERAL_QUOTE</a>
+    production allows the use of unescaped control characters.
+    Although this specification does not directly expose this content to an end user,
+    it might be presented through 
+    If presented through a user agent, this may cause the </p>
```

Comment:
```suggestion
    Although this specification does not directly expose this content to an end user,
    it might be presented through a user agent, this may cause the presented text to be obfuscated due to presentation of such characters.</p>
```

Reply:
```suggestion
    Although this specification does not directly expose this content to an end user,
    it might be presented through a user agent, which may cause the presented text to 
    be obfuscated due to presentation of such characters.</p>
```